docs/root/module-list.md

```diff
@@ -0,0 +1,6 @@
+```{toctree}
```

Comment:
This fixes a broken link issueswhere if someone visits `<doc URI>/modules` then they will get a 404